impeller/entity/geometry/point_field_geometry.cc

```diff
@@ -27,14 +27,29 @@ GeometryResult PointFieldGeometry::GetPositionBuffer(
     return {};
   }
   const Matrix& transform = entity.GetTransform();
+  HostBuffer& host_buffer = renderer.GetTransientsBuffer();
+
   Scalar max_basis = transform.GetMaxBasisLengthXY();
   if (max_basis == 0) {
     return {};
   }
 
   Scalar min_size = 0.5f / max_basis;
   Scalar radius = std::max(radius_, min_size);
-  HostBuffer& host_buffer = renderer.GetTransientsBuffer();
+
+  if (radius_ <= min_size) {
```

Comment:
It looks like this is 1/2 a pixel - is that the cutoff we want?

Also, this test makes the std::max above pointless. We could replace the ::max with just whether this test triggers (unless we change the cutoff to be different than 0.5 pixels?)

Reply:
Radius of 1/2 a pixel so cut off at 1 pixel size makes sense to me. Removed max.

Reply:
D'oh! Radius.